src/layman/geoserver_proxy.py

```diff
@@ -204,35 +204,49 @@ def proxy(subpath):
 
     # add buffer if incoming request is WMS GetMap in EPSG:3857 and one of SLD layers has native CRS EPSG:5514
     # otherwise features are missing in the response in this specific case, we are not sure about the reason
-    if query_params.get('service') == 'WMS' and query_params.get('request') == 'GetMap' \
-            and (query_params.get('crs') or query_params.get('srs')) == crs_def.EPSG_3857:
+    if query_params.get('service') == 'WMS' and query_params.get('request') == 'GetMap':
```

Comment:
I prefer to limit this branch as much and as soon as possible, because there used to be a lot of frequent GetMap requests and I want to save requests to database, and because it's more readable. So I suggest to use also the second part of condition like
`and (query_params.get('crs') or query_params.get('srs')) in {crs_def.EPSG_3857, crs_def.CRS_84}`